ballerina/Ballerina.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 org = "ballerina"
 name = "xmldata"
-version = "2.0.1"
+version = "2.0.2"
```

Comment:
Let's do minor version bump as we are adding new feature

Reply:
Bumped to `2.1.0`